etcdserver/metrics.go

```diff
@@ -76,6 +76,12 @@ var (
 		Name:      "slow_apply_total",
 		Help:      "The total number of slow apply requests (likely overloaded from slow disk).",
 	})
+	applySnapshotInflights = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd",
+		Subsystem: "server",
+		Name:      "snapshot_apply_inflights_total",
```

Comment:
nit: Since this is not a network metric (and subsystem=server), maybe instead of "inflight" name it something like "snapshot_apply_in_progress_total" ?